site/includes/deptfeatures-wet-126.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="gc-prtts mrgn-tp-md">
+	<div class="container">
+		<div class="row">
+			<section class="provisonal col-md-8 mrgn-bttm-md mrgn-tp-lg">
+				<h2 class="wb-inv">Feature</h2>
+				<a href="#">
+					<figure class="row">
```

Comment:
This is the old pattern for the contextual feature, now we leverage the `.stretchlink` CSS